clickhouse-core/src/main/scala/xenon/clickhouse/hash/Util.scala

```diff
@@ -0,0 +1,55 @@
+package xenon.clickhouse.hash
+
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+object Util {
```

Comment:
HashUtils

Reply:
I thought it is under hash package, and repeating Hash would be redundant. However it should be not major issue

Reply:
But it is not easy to distinguish on the caller's side
```
import xenon.clickhouse.hash.Util
...
Util.xxxx
```

Reply:
That makes sense. Fixed that